hazelcast/src/main/java/com/hazelcast/config/HotRestartPersistenceConfig.java

```diff
@@ -90,6 +98,21 @@ public HotRestartPersistenceConfig setBaseDir(File baseDir) {
     }
 
     /**
+     * Gets the configured number of Hot Restart store instance to create for one Hazelcast instance.
+     */
+    public int getParallelism() {
+        return parallelism;
+    }
+
+    /**
+     * Sets the number of Hot Restart store instances to create for one Hazelcast instance.
+     */
+    public HotRestartPersistenceConfig setParallelism(int parallelism) {
```

Comment:
OK, then basically repeat what XML Schema validates, because this can be reached programmatically as well.